### PR TITLE
Add guidance to avoid future predictions

### PR DESCRIPTION
**Pull request by bburt-rh**
Draft of content to provide guidance for referring to future releases, plans, or events. 

Addresses issue: https://github.com/redhat-documentation/supplementary-style-guide/issues/51

Please comment and provide feedback.

### CODE REVIEW

File: supplementary_style_guide/style_guidelines/statements-about-the-future.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- Never promise delivery of a fix, feature, or release on a specific timeline.
+- Following the guidelines in the _IBM Style Guide_, do not refer to a collective "we" when referring to Red Hat. 
+
+.Example: Incorrect usage
+`We will fix this issue in the 18.3.4 release next February.`
+
+.Example: Correct usage
+`It is anticipated that an upcoming release will include a fix for this issue.`
+
+.Additional resources
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/statements-about-the-future.adoc:24]: This is an external guide, so you'll have to remove this link

File: supplementary_style_guide/master.adoc [hunk 2]
```diff
@@ -40,6 +40,9 @@ include::style_guidelines/specific-doc-types.adoc[leveloffset=+2]
 // Links
 include::style_guidelines/links.adoc[leveloffset=+2]
 
+// Statements about the future
```
Comment (bergerhoffer) [inline supplementary_style_guide/master.adoc:43]: Since this really is only a single thing and not necessarily a category that will have additional guidelines added to it, I think we probably don't need to make its own category. I'd recommend just adding it to the "General formatting" section's file, kind of like how the conscious language one is there.

File: supplementary_style_guide/style_guidelines/statements-about-the-future.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+= Statements about the future
+
+[[making-statements-about-future-releases-or-plans]]
+== Making statements about future releases or plans
+
+Regarding future product releases, the _IBM Style Guide_ states, "Avoid claims that are related to the content or timing of a future product or release."  
+
+However, under some limited circumstances (such as _Release Notes_), circumstances will dictate that you refer to a future release, plan, or event. 
+In these situations, follow these guidelines:
+
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/statements-about-the-future.adoc:11]: Can you use asterisks for the bulleted list below? I believe that's preferred

File: supplementary_style_guide/style_guidelines/statements-about-the-future.adoc [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+- When discussing future plans, use words such as "anticipate," "expect" or "plan."
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/statements-about-the-future.adoc:12]: Can you put the period and the comma outside of the ending quote? That's a new ISG rule

File: supplementary_style_guide/style_guidelines/statements-about-the-future.adoc [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+- Do not make any promises that a fix for a known issue will ever be included or that a specific feature will be added in an upcoming release.
+- Never refer to a specific future release. Do not refer to a release number or a specific release timeline. 
+- Never promise delivery of a fix, feature, or release on a specific timeline.
+- Following the guidelines in the _IBM Style Guide_, do not refer to a collective "we" when referring to Red Hat.
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/statements-about-the-future.adoc:16]: This is already covered in ISG and elsewhere in the SSG. I don't think it's necessary for this section

File: supplementary_style_guide/style_guidelines/statements-about-the-future.adoc [hunk 6]
```diff
@@ -0,0 +1,25 @@
+[[statements-about-the-future]]
+= Statements about the future
+
+[[making-statements-about-future-releases-or-plans]]
+== Making statements about future releases or plans
+
+Regarding future product releases, the _IBM Style Guide_ states, "Avoid claims that are related to the content or timing of a future product or release."
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/statements-about-the-future.adoc:7]: I don't think that we'd want to state ISG's guidelines word for word. I'd try to reword. Maybe something like

```
When possible, avoid making statements that predict future releases or plans.
```

File: supplementary_style_guide/style_guidelines/statements-about-the-future.adoc [hunk 7]
```diff
@@ -0,0 +1,25 @@
+[[statements-about-the-future]]
+= Statements about the future
+
+[[making-statements-about-future-releases-or-plans]]
+== Making statements about future releases or plans
+
+Regarding future product releases, the _IBM Style Guide_ states, "Avoid claims that are related to the content or timing of a future product or release."  
+
+However, under some limited circumstances (such as _Release Notes_), circumstances will dictate that you refer to a future release, plan, or event.
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/statements-about-the-future.adoc:9]: You have "circumstances" twice here. Maybe reword something like:

```
However, some circumstances, such as release notes or deprecations, might dictate that you refer to a future release, plan, or event. 
```

File: supplementary_style_guide/style_guidelines/statements-about-the-future.adoc [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+== Making statements about future releases or plans
+
+Regarding future product releases, the _IBM Style Guide_ states, "Avoid claims that are related to the content or timing of a future product or release."  
+
+However, under some limited circumstances (such as _Release Notes_), circumstances will dictate that you refer to a future release, plan, or event. 
+In these situations, follow these guidelines:
+
+- When discussing future plans, use words such as "anticipate," "expect" or "plan."
+- Do not make any promises that a fix for a known issue will ever be included or that a specific feature will be added in an upcoming release.
+- Never refer to a specific future release. Do not refer to a release number or a specific release timeline.
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/statements-about-the-future.adoc:14]: Instead of "Never", perhaps simply "Do not" here and in the below bullet?

**Review by bergerhoffer — COMMENTED** [review]
Thanks a lot for tracking this down and submitting the PR @bburt-rh! Left a few comments

File: supplementary_style_guide/style_guidelines/statements-about-the-future.adoc (hunk 1 above)
Comment (bburt-rh) [reply supplementary_style_guide/style_guidelines/statements-about-the-future.adoc]: done

File: supplementary_style_guide/master.adoc (hunk 2 above)
Comment (bburt-rh) [reply supplementary_style_guide/master.adoc]: done

File: supplementary_style_guide/style_guidelines/statements-about-the-future.adoc (hunk 3 above)
Comment (bburt-rh) [reply supplementary_style_guide/style_guidelines/statements-about-the-future.adoc]: done

File: supplementary_style_guide/style_guidelines/statements-about-the-future.adoc (hunk 4 above)
Comment (bburt-rh) [reply supplementary_style_guide/style_guidelines/statements-about-the-future.adoc]: done

File: supplementary_style_guide/style_guidelines/statements-about-the-future.adoc (hunk 5 above)
Comment (bburt-rh) [reply supplementary_style_guide/style_guidelines/statements-about-the-future.adoc]: removed

File: supplementary_style_guide/style_guidelines/statements-about-the-future.adoc (hunk 7 above)
Comment (bburt-rh) [reply supplementary_style_guide/style_guidelines/statements-about-the-future.adoc]: fixed

File: supplementary_style_guide/style_guidelines/statements-about-the-future.adoc (hunk 8 above)
Comment (bburt-rh) [reply supplementary_style_guide/style_guidelines/statements-about-the-future.adoc]: done

**Review by ndeevy — DISMISSED** [review]

**Comment by ndeevy** [comment]
@bburt-rh  This looks perfect to me!

File: supplementary_style_guide/style_guidelines/general-formatting.adoc [hunk 9]
```diff
@@ -268,3 +268,23 @@ For example, you might combine existing product name attributes to create compou
 :name-runtime-vertx: Eclipse Vert.x
 :name-spring-reactive: {name-runtime-spring-boot} with {name-runtime-vertx} reactive components
 ----
+
+[[statements-about-the-future]]
+== Statements about the future
+
+=== Making statements about future releases or plans
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:275]: Since there's only one section, I don't think you need both of these headings. I'd probably remove this one (or incorporate the wording into the above heading, "Statements about future releases or plans" or something like that)

**Review by bergerhoffer — APPROVED** [review]
Just one more non-critical comment. LGTM, thanks for the updates!

File: supplementary_style_guide/style_guidelines/general-formatting.adoc (hunk 9 above)
Comment (bburt-rh) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:275]: @bergerhoffer Good point. That makes sense. I'll do a quick fix on it right now.

Comment (bburt-rh) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc]: @bergerhoffer Just pushed the changes out. PTAL.

**Review by bergerhoffer — APPROVED** [review]
Thanks, LGTM!

@redhat-documentation/ccs-style-council need at least one more review/approval if someone can take a look

**Review by ndeevy — APPROVED** [review]

**Review by stoobie — APPROVED** [review]
LGTM
